prepare-videochat.sh

```diff
@@ -263,25 +268,25 @@ if ! has_kernel_module v4l2loopback; then
             info "Installation failed.  Please install v4l2loopback manually from github.com/umlaeute/v4l2loopback."
         fi
     elif [ $DIST = "Arch" ]; then
-        yaourt -S v4l2loopback-dkms
+        error "Plesae install v4l2loopback-dkms package (available in AUR)"
     fi
 
     if has_kernel_module v4l2loopback; then
         info "The v4l2loopback kernel module was installed successfully."
     else
-        error "Could not install the v4l2loopback kernel module through apt-get or yaourt."
+        error "Could not install the v4l2loopback kernel module through apt-get."
     fi
 fi
 
 # check if the user has the pulse gst plugin installed
-if find "/usr/lib/gstreamer-$GST_VER/libgstpulse.so" | egrep -q '.*'; then # lc
+if find "/usr/lib/gstreamer-$GST_VER/libgstpulse.so" | egrep -q '.*'; then
     # plugin installed, do nothing
     info "Found the pulse gst plugin"
 else
     if [ $DIST = "Debian" ] || [ $DIST = "Ubuntu" ]; then
         sudo apt-get install -y gstreamer${GST_VER}-pulseaudio
-    elif can_run yaourt; then
-        error "we should figure out what package supplies this"
```

Comment:
You mean you want to tell user what package to install? Package's name is gst-plugins-good in Arch Linux. This message is already in my commit.
Or did you mean anything else?